rook/initialise.go

```diff
@@ -465,3 +479,24 @@ func doTemplate(input string, answers Answers) (output string, err error) {
 	output = out.String()
 	return
 }
+
+func generateEntryCode(filename string, entryType string) (err error) {
+	buf := bytes.Buffer{}
+	funcName := "main()"
```

Comment:
It's probably best you also add `OnGameModeInit` for gamemodes, since it is the counter part to `OnFilterScriptInit` 
this isn't required, but if you feel up to trying to add it, that would be good.

Reply:
Okay, i'll try to add it later when i deal with the part first, thanks ^^